app/javascript/crayons/MultiInput/__stories__/MultiInput.mdx

```diff
@@ -1 +1,32 @@
-# Multi-Input
\ No newline at end of file
+# Multi-Input
+
+The MultiInput component can be used in situations where a user wants to capture/add multiple entries.
+It allows the ability to add, edit, remove and capture and validate each entry before submitting the form.
+
+## Adding an entry
+
+Once a user types their entry, they may submit it by pressing Enter, Comma or Space.
+
+## Editing and removing an entry
+
+When a user adds an entry, it will be displayed as a button group. The first button triggers an edit, and the second button triggers the removal of the entry.
+
+Both buttons appear in the tab order, and are semantically grouped together for assistive technology.
+
+## Technical implementation notes
+
+### Styling
+
+The component comes with some default styling for selected options, but the layout of selections can be customised by passing in a Preact component `SelectionTemplate`.
+
+If the template is passed, it will receive in props the full option object in props e.g. `{ name: 'option one', someOtherProperty: 'foo' }`.
+
+### Accessibility
+
+The component follows the [WAI-ARIA Combobox authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/#combobox) to ensure operability for a wide range of users.
```

Comment:
Combobox is only for "type and suggest" UI like the autocomplete, so I think not applicable here (we didn't add `role="combobox` for this reason 😄)